### PR TITLE
Consistency updates

### PR DESCRIPTION
**Pull request by BasileiosKal**
Changes for consistency:
1. `len()` -> `length()`.
2. Consistent array de-structuring.
3. Make code blocks < 72 chars.
4. Update `create_generators`. Same result, removed the `while` and added a `go back to step 4` for consistency (we where referencing `generator_0` without defining it at the first step where `i = 1`).

### CODE REVIEW

**Comment by BasileiosKal** [comment]
Discussed on the WG call. The issue of the naming convention for SK and PKs came up (neither the SK not the PK is a point to be capitalized). Will address to another PR to avoid conflicts with #186.

File: draft-bbs-signatures.md [hunk 1]
```diff
@@ -1603,12 +1614,12 @@ Along with the PK value as defined in (#key-pair) as inputs into the Verify oper
 The following section provides an explanation of how the ProofGen and ProofVerify operations work.
 
 Let the prover be in possession of a BBS signature `(A, e, s)` with `A = B * (1/(e + Sk))` where `Sk` the signer's secret key and,
```
Comment (andrewwhitehead) [inline draft-bbs-signatures.md:1616]:
```suggestion
Let the prover be in possession of a BBS signature `(A, e, s)` with `A = B * (1/(e + SK))` where `SK` the signer's secret key and,
```

**Comment by BasileiosKal** [comment]
Also closes #113

File: draft-bbs-signatures.md [hunk 2]
```diff
@@ -1603,12 +1614,12 @@ Along with the PK value as defined in (#key-pair) as inputs into the Verify oper
...
+```
+B = P1 + h0 * s + h[1] * msg_1 + ... + h[L] * msg_L
```
[the remark this answers is not in the document]
Comment (BasileiosKal) [reply draft-bbs-signatures.md]: Yea! Good catch. Updated to use `H_s, H_d` add added `domain` (not explaining what it is, using it like `e, s` etc).

File: draft-bbs-signatures.md [hunk 3]
```diff
@@ -1602,36 +1613,36 @@ Along with the PK value as defined in (#key-pair) as inputs into the Verify oper
...
 
-  If both EQ1 and EQ2 hold, and `e(A', Pk) = e(Abar, P2)`, an extractor can return a valid BBS signature from the signers `Sk`, on the disclosed messages. The proof returned is `(A', Abar, d, nizk)`. To validate the proof, a verifier checks that `e(A', Pk) = e(Abar, P2)` and verifies `nizk`.
+Note that the verifier will know the elements in the left side of the above equations (i.e., `C1` and `C2`) but not in the right side (i.e., `s'`, `r3` and the undisclosed messages: `msg_j1, ..., msg_jU`). However, using the `nizk`, the prover can convince the verifier that they (the prover) know the elements that satisfy those equations, without disclosing them. Then, if both EQ1 and EQ2 hold, and `e(A', Pk) = e(Abar, P2)`, an extractor can return a valid BBS signature from the signer's `SK`, on the disclosed messages. The proof returned is `(A', Abar, D, nizk)`. To validate the proof, a verifier checks that `e(A', PK) = e(Abar, P2)` and verifies the `nizk`. Validating the proof, will guarantee the authenticity and integrity of the disclosed messages, as well as ownership of the undisclosed messages and of the signature.
```
Comment (andrewwhitehead) [inline draft-bbs-signatures.md:1645]:
```suggestion
Note that the verifier will know the elements in the left side of the above equations (i.e., `C1` and `C2`) but not in the right side (i.e., `s'`, `r3` and the undisclosed messages: `msg_j1, ..., msg_jU`). However, using the `nizk`, the prover can convince the verifier that they (the prover) know the elements that satisfy those equations, without disclosing them. Then, if both EQ1 and EQ2 hold, and `e(A', PK) = e(Abar, P2)`, an extractor can return a valid BBS signature from the signer's `SK`, on the disclosed messages. The proof returned is `(A', Abar, D, nizk)`. To validate the proof, a verifier checks that `e(A', PK) = e(Abar, P2)` and verifies the `nizk`. Validating the proof, will guarantee the authenticity and integrity of the disclosed messages, as well as ownership of the undisclosed messages and of the signature.
```

Comment (BasileiosKal) [reply draft-bbs-signatures.md:1645]: Thank you!!

**Review by tplooker — APPROVED** [review]

**Comment by tplooker** [comment]
Multiple approvals, purely editorial, merging

**Pull request merged**